src/js/components/FormField/FormField.js

```diff
@@ -179,26 +180,30 @@ const FormField = forwardRef(
       );
     }
 
-    const contentProps = { ...formFieldTheme.content };
+    const fieldContentProps = { ...formFieldTheme.content };
 
     if (!pad && !wantContentPad) {
-      contentProps.pad = undefined;
+      fieldContentProps.pad = undefined;
     }
 
     if (themeBorder && themeBorder.position === 'inner') {
       if (error && formFieldTheme.error) {
-        contentProps.background = formFieldTheme.error.background;
+        fieldContentProps.background = formFieldTheme.error.background;
       } else if (disabled && formFieldTheme.disabled) {
-        contentProps.background = formFieldTheme.disabled.background;
+        fieldContentProps.background = formFieldTheme.disabled.background;
       }
     }
-    // Removing margin from contentProps because margin should be applied
+    // Removing margin from fieldContentProps because margin should be applied
     // on containing <FormFieldContentBox>
-    // contentProps.margin = undefined;
-    // contents = <Box {...contentProps}>{contents}</Box>;
+    // fieldContentProps.margin = undefined;
+    // contents = <Box {...fieldContentProps}>{contents}</Box>;
```

Comment:
@MikeKingdom I believe these should be removed altogether. I missed removing these when resolving a merge conflict two weeks back. I can handle in a separate PR if you prefer.

Reply:
> @MikeKingdom I believe these should be removed altogether. I missed removing these when resolving a merge conflict two weeks back. I can handle in a separate PR if you prefer.

@MikeKingdom are you okay with removing these commented lines? Or would you like me to in a separate PR?

Reply:
I can, especially since I'm making other tweaks based on feedback. Sorry I missed this in the most recent changes

Reply:
removed